ui/push-health/TestFailure.jsx

```diff
@@ -31,24 +31,39 @@ export default class TestFailure extends React.PureComponent {
           </span>
         </Row>
         <div className="small">
-          <a
-            className="text-dark ml-3 px-1 border border-secondary rounded"
-            href={getJobsUrl({ selectedJob: jobId, repo, revision })}
-          >
-            {jobName}
-          </a>
-          <span className="ml-1">{config},</span>
-          <span className="ml-1">
-            {classification !== 'not classified' && (
-              <FontAwesomeIcon icon={faStar} />
-            )}
-            <span className="ml-1">{classification}</span>
+          <span>
+            {platform} {config}:
           </span>
+          {jobs.map(job => (
+            <span className="mr-2">
+              <a
+                className="text-dark ml-3 px-1 border border-secondary rounded"
+                href={getJobsUrl({ selectedJob: job.id, repo, revision })}
+                title={jobName}
+              >
+                {jobSymbol}
```

Comment:
Converted this to showing the job symbol like in Treeherder, and the job name in the hover text so it wasn't so busy, especially with multiple jobs.